code/game/jobs/job/science.dm

```diff
@@ -193,3 +193,33 @@
 	satchel = /obj/item/storage/backpack/satchel_tox
 	dufflebag = /obj/item/storage/backpack/duffel/tox
 	messengerbag = /obj/item/storage/backpack/messenger/tox
+
+/datum/job/officer/sci
+	title = "Science Officer"
+	flag = SCISEC
+	department = "Science"
+	department_flag = MEDSCI
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the research director and head of security"
+	selection_color = "#e038e0"
+	economic_modifier = 4
+
+	minimum_character_age = 18
+
+	access = list(access_security, access_brig, access_sec_doors, access_eva, access_external_airlocks, access_weapons, access_tox, access_tox_storage, access_research, access_robotics, access_xenobiology, access_xenoarch, access_maint_tunnels)
```

Comment:
Robotics, R&D, Xenobiology, EVA.
This is the unga antag RD without  the whitelist + brig access, why exactly is this a thing?
The PR is handing out double the access for the strongest antags, also being the most combat equipped players.
Is this really a smart idea?

Reply:
I agree with you on this, but other developers that I collaborated on with the matter disagreed and outnumbered me, so my hands are tied in this area.